cabot/cabotapp/schedule_validation.py

```diff
@@ -55,18 +56,33 @@ def _find_problems(schedule, current_time=None):
     * Is there a fallback officer? Do they have an email set?
     * Is the schedule empty?
     * Are there any gaps between the events in the schedule (where no one is on call)?
+    * Is the schedule iCal url valid (returns an HTTP 200 and can be parsed)?
     :param schedule: schedule to check (should already be updated from update_shifts())
     :param current_time: time to start checking the schedule from (to see if anyone is currently on call),
                          None for now
     :return: list of strings describing any problems in the schedule ("problems")
     """
     problems = []
 
+    # check for a valid fallback officer
     if not schedule.fallback_officer:
         problems.append("The schedule has no fallback officer.")
     elif not schedule.fallback_officer.email:
         problems.append("The fallback officer does not have an email set.")
 
+    # test the iCal url
+    if schedule.ical_url:
+        try:
+            schedule.get_calendar_data()
+        except requests.RequestException as e:
+            problems.append("The schedule's iCal URL returns an error ({}).".format(e))
```

Comment:
nit: maybe "The schedule's iCal URL is invalid"? "returns an error" is kinda confusing imo